aicssegmentation/structure_wrapper_config/all_functions.json

```diff
@@ -5,15 +5,15 @@
         "parameter": {
             "scaling_param": [
                 {
-                    "widge type": "slider",
-                    "data type": "float",
+                    "widget_type": "slider",
```

Comment:
This `widget_type` isn't exposed anywhere in the code - are you planning to delegate that UI aspect of things to the plugin as discussed? If so this can probably be deleted

Reply:
One option is turning widget-type into an enum that is exported from aics-segmentation